src/cbc_sdk/platform/alerts.py

```diff
@@ -1411,3 +1430,45 @@ def set_remote_is_private(self, is_private):
         """
         self._criteria["remote_is_private"] = is_private
         return self
+
+    def add_exclusion_threat_notes_present(self, is_present):
```

Comment:
I'm tempted to re-used are existing settters and make an optional property for exclusion instead of duplicating the functions and docstrings

Reply:
done